src/IBM.WatsonDeveloperCloud/Http/Request.cs

```diff
@@ -109,6 +109,7 @@ public TaskAwaiter<IResponse> GetAwaiter()
         public async Task<T> As<T>()
         {
             HttpResponseMessage message = await this.AsMessage().ConfigureAwait(false);
+            var result = message.Content.ReadAsStringAsync().Result;

```

Comment:
@atilatosta I was trying this. How can I get access to the request when calling the service?

```
private void ClassifyGet()
{
    var result = _visualRecognition.Classify(_imageUrl);
}
```